de.tu_bs.cs.isf.e4cf.compare.data_structures/src/de/tu_bs/cs/isf/e4cf/compare/data_structures/io/reader/JavaNodeTypes.java

```diff
@@ -0,0 +1,28 @@
+package de.tu_bs.cs.isf.e4cf.compare.data_structures.io.reader;
+
+/**
+ * This file extends the nodes types and attributes for the generic data structure generated by @see Visitor 
+ * 
+ * @author Serkan Acar
+ * @author Pascal Blum
+ * @author Paulo Haas
+ * @author Hassan Smaoui
+ */
+public enum JavaNodeTypes {
+	Argument,
+	Assignment,
+	BlockComment,
+	Body,
+	Bound,
+	Break,
+	Cast,
+	Class,
+	Continue, 
+	Else,

```

Comment:
Else is in the Attributes as well

Reply:
They have different purporses.  The attribute is used by conditional expr, where the else expr can be added as an attribute; the node type is used for else statement, which can have more complex bodies, that cannot be simply added as an attribute.
Both files now have Javadocs.